service/controller/cluster.go

```diff
@@ -160,10 +160,8 @@ func newClusterResourceSets(config ClusterConfig) ([]*controller.ResourceSet, er
 	{
 		c := clusterResourceSetConfig{
 			CertsSearcher:      certsSearcher,
-			CtrlClient:         config.K8sClient.CtrlClient(),
-			G8sClient:          config.K8sClient.G8sClient(),
 			HostAWSConfig:      config.HostAWSConfig,
-			K8sClient:          config.K8sClient.K8sClient(),
+			K8sClient:          config.K8sClient,
```

Comment:
I am wiring the client collection now as this is easier to work with down the road.